### PR TITLE
Add Docusaurus

### PR DESCRIPTION
**Pull request by JosipJanus**
This PR covers #14.

### CODE REVIEW

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      - main
+jobs:
+  build-and-deploy:
+    concurrency: ci-${{ github.ref }} # Recommended if you intend to make multiple deployments in quick succession.
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2
+
+      - name: Install and Build 🔧 # This example project is built using npm and outputs the result to the 'build' folder. Replace with the commands required to build your project, or remove this step entirely if your site is pre-built.
```
Comment (fvoska) [inline .github/workflows/docs.yml:14]: Remove this placeholder comment

File: ngx-nuts-and-bolts-docs/docs/loading-state/loading-state.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
+---
+id: component-with-loading-state
+title: Component with loading state
+sidebar_label: Component with loading state
+---
+
+# Component with loading state
```
Comment (fvoska) [inline ngx-nuts-and-bolts-docs/docs/loading-state/loading-state.md:7]: Remember to update this after #17 is merged because this readme file was updated.

**Review by fvoska — CHANGES_REQUESTED** [review]

File: ngx-nuts-and-bolts-docs/docs/loading-state/loading-state.md (hunk 2 above)
Comment (JosipJanus) [reply ngx-nuts-and-bolts-docs/docs/loading-state/loading-state.md:7]: Sure, this was only for testing purposes

**Review by fvoska — APPROVED** [review]
Let's see if this works!

**Pull request merged**